apps/store/themes/store/js/categorization-data.js

```diff
@@ -23,42 +23,62 @@
         if (typeof history.onpushstate == "function") {
             history.onpushstate({state: state});
         }
-        loadCategorizationEntries(url);
+        if (!state || state.categorization == false) {
+            loadCategorizationEntries(url);
+        }
+        loadTagEntries(url);
         return pushState.apply(history, arguments);
     };
+})(window.history);
 
-    /**
-     * This method loads the categorization-entry partial data when selection query changes.
-     *
-     * @param url   url to load the data from
-     */
-    function loadCategorizationEntries(url) {
-        caramel.data({
-            title: null,
-            body: ['assets']
-        }, {
-            url: url,
-            success: function (data, status, xhr) {
-                caramel.partials(data._.partials, function () {
-                    caramel.render('categorization-entry', data.body.assets.context, function (info, content) {
-                        $('#categorization-div').html(content);
-                        categorization();
-                        $('.refine > .panel > div').first().collapse('show');
-                    });
+/**
+ * This method loads the categorization-entry partial data when selection query changes. *
+ * @param url   url to load the data from
+ */
+function loadCategorizationEntries(url) {
+    caramel.data({
+        title: null,
```

Comment:
@splinter Thanks for pointing out. Removed this in #841 (c7e7f2fba36a308ff02c339614352b54567c54fa)